### ISSUE TITLE
Investigate path length limit on Windows

### ISSUE THREAD

**Issue by Max-7** [Scille/parsec-cloud#928]
And the errors it can create when syncing file from Linux.

**Comment by Max-7** [comment]
I created a ~300 characters path (3 folders of ~100 characters each) on Linux. In it, I put a 200 characters long file (a .txt file) and a few characters long file (the "control group").

No problem on Linux, I've tried a few apps and all of them could open the file.

On Windows, the Parsec GUI acts as usual, and even the Windows Explorer seems to have no trouble displaying large folder names. Errors arise when trying to perform more challenging tasks.

Opening the file leads to different results depending on the app.
For exemple, with Notepad++:
- when trying to open the file by double-clicking on it (.txt are open by default with it), a dialog from EXPLORE.exe appears and warns that the directory name is invalid, nothing else is done).
- when trying to open using the context menu (right click -> Edit with Notepad++), nothing happens.
- when trying to open using the Windows context menu (right click -> Open With...), nothing happens.
- when trying to open it from Notepad++ (menu File/Open), nothing happens

Other apps (Microsoft Word, LibreOffice, WordPad, Acrobat Reader, Chrome, ...) have the same kind of issues (sometimes signaling that the file is not found/can not be opened)

Ironically, the good old Notepad on Windows has absolutely no trouble opening and editing the file at all.

Trying with the file with a short name leads to the same results.

Windows Explorer, while it's able to display the file, can not do anything else. It's impossible to copy/paste a file inside the folder, rename a file or copy a file already in the folder. These operations fail without anything happening (not even an error dialog). It is possible though to copy a file from the folder to another destination.

The GUI works perfectly fine and does all of these operations as expected.

It seems perfectly possible to create an app on Windows handling long file names (don't know the real limit but it's probably given by the file system, see https://en.wikipedia.org/wiki/Filename#Comparison_of_filename_limitations); and indeed, Parsec, WinFSP, and even Notepad don't seem to have problems, but most apps won't be able to open the file, and even the Windows Explorer will have problem getting anything done (other than displaying the files).

These problems could be perceived by a user as being Parsec's fault (especially since Windows Explorer and most other apps do not even display an error message).
